File: comparisons/mapreduce/mapreduce.cs
using System;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        var somaDosParesAoQuadrado =
            Enumerable.Range(1, 100)
                .Where(x => x % 2 == 0)
                .Select(x => x * x)
                .Aggregate((acc, x) => acc + x);

        Console.WriteLine(somaDosParesAoQuadrado);
    }
}

File: comparisons/mapreduce/mapreduce.vb
Module mapreduce

    Sub Main()

        Dim somaDosParesAoQuadrado =
            Enumerable.Range(1, 100) _
                .Where(Function(x) x Mod 2 = 0) _
                .Select(Function(x) x ^ 2) _
                .Aggregate(Function(acc, x) acc + x)

        Console.WriteLine(somaDosParesAoQuadrado)

    End Sub

End Module
